### PR TITLE
NMS-12695: graphite telemetry adapter

### PR DESCRIPTION
**Pull request by RangerRick**
This PR adds basic support for the Graphite `plaintext` protocol to Telemetryd.

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-12695
* Bamboo (Continuous Integration): https://bamboo.opennms.org/

### CODE REVIEW

**Comment by cgorantla** [comment]
Looks good to me. But if this not core network telemetry we should find a way to integrate this through integration api and make it available as plugin that we can load into OpenNMS

**Comment by j-white** [comment]
> But if this not core network telemetry we should find a way to integrate this through integration api and make it available as plugin that we can load into OpenNMS

This can be "core" - the idea is to have an easy way to get arbitrary metrics in that aren't tied to specific vendor implementations.

**Comment by cgorantla** [comment]
> > But if this not core network telemetry we should find a way to integrate this through integration api and make it available as plugin that we can load into OpenNMS
> 
> This can be "core" - the idea is to have an easy way to get arbitrary metrics in that aren't tied to specific vendor implementations.

:+1:

File: features/telemetry/protocols/graphite/adapter/src/main/java/org/opennms/netmgt/telemetry/protocols/graphite/adapter/GraphiteAdapter.java [hunk 1]
```diff
@@ -95,12 +96,12 @@ public GraphiteAdapter(final AdapterDefinition adapterConfig, final MetricRegist
             if (entry.length != 3) {
                 LOG.warn("Unparseable graphite plaintext message: {}", line);
             } else {
-                final GraphiteMetric metric = new GraphiteMetric(entry[0], entry[1], Long.valueOf(entry[2], 10));
                 try {
+                    final GraphiteMetric metric = new GraphiteMetric(entry[0], entry[1], Long.valueOf(entry[2], 10));
```
Comment (j-white) [inline features/telemetry/protocols/graphite/adapter/src/main/java/org/opennms/netmgt/telemetry/protocols/graphite/adapter/GraphiteAdapter.java:100]: The `NumberFormatException`  still won't get caught here right?

Comment (RangerRick) [reply features/telemetry/protocols/graphite/adapter/src/main/java/org/opennms/netmgt/telemetry/protocols/graphite/adapter/GraphiteAdapter.java:100]: d'oh, you're right

**Pull request merged**